build.py

```diff
@@ -95,6 +105,33 @@
 wheels_dir = os.path.join(root_dir, "target", "wheels")
 vscode_src = os.path.join(root_dir, "vscode")
 jupyterlab_src = os.path.join(root_dir, "jupyterlab")
+package_name = "qsharp-nightly"
+
+if os.environ.get("QSHARP_PACKAGE_SUFFIX") is not None:
```

Comment:
Could you add a comment explaining _who_ might define these variables (this and `QSHARP_PACKAGE_VERSION`)? A build pipeline?